examples/factors/README.md

```diff
@@ -83,38 +81,46 @@ fn main() {
     let b: u64 = 23;
 ```
 
-Currently, our host driver program creates and runs a prover.
-The `prover.run()` command will cause our guest program to execute:
+Currently, our host driver program creates an executor environment before constructing an executor. The `Executor::run()` command will produce a session. The `Session::prove()` command will cause our guest program to execute, producing a receipt:
 
 ```rust
-    // Make the prover.
-    let mut prover =
-        Prover::new(MULTIPLY_ELF).expect("Prover should be constructed from valid ELF binary");
+    // First, we construct an executor environment
+    let env = ExecutorEnv::default();
+
+    // TODO: add guest input to the executor environment using
+    // ExecutorEnv::add_input()
```

Comment:
If this is also wrong in the in-code comment, that will need to be updated as well.

```suggestion
    // ExecutorEnvBuilder::add_input()
```